com.itopia.client.json

```diff
@@ -0,0 +1,241 @@
+{
+    "app-id": "com.itopia.client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "itopia Remote Desktop Client",
+    "only-arches": [
```

Comment:
If you wish to distribute your application for x86_64 only, you need to add a `flathub.json` file to limit the builders to that arch as well, see https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64 and remove the key from here as it won't work.

Reply:
@bilelmoussaoui oh, sorry, fixed that. did miss that this required an extra `flathub.json` file.